### PR TITLE
Search part 3

### PR DESCRIPTION
**Pull request by JohnOberhauser**
- Setting up network call
- Setting up database tables

### CODE REVIEW

File: core/database/src/main/java/org/mozilla/social/core/database/model/entities/hashtagCollections/SearchedHashTag.kt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+data class SearchedHashTagWrapper(
+    @Embedded
+    val searchedHashTag: SearchedHashTag,
+
+    @Relation(
+        parentColumn = "hashTagName",
+        entityColumn = "name",
+    )
+    val status: DatabaseHashTagEntity,
```
Comment (devotaaabel) [inline core/database/src/main/java/org/mozilla/social/core/database/model/entities/hashtagCollections/SearchedHashTag.kt:36]: Is this supposed to be called `status`?

**Review by devotaaabel — APPROVED** [review]
Excited for the app to have search 🔥

File: core/database/src/main/java/org/mozilla/social/core/database/model/entities/hashtagCollections/SearchedHashTag.kt (hunk 1 above)
Comment (JohnOberhauser) [reply core/database/src/main/java/org/mozilla/social/core/database/model/entities/hashtagCollections/SearchedHashTag.kt:36]: oops

Comment (JohnOberhauser) [reply core/database/src/main/java/org/mozilla/social/core/database/model/entities/hashtagCollections/SearchedHashTag.kt:36]: I'm just going to fix it in the next PR

**Pull request merged**